counting_words.rb

```diff
@@ -0,0 +1,20 @@
+def get_occurences(words)
```

Comment:
good stuff. obviously provided more information, you might... 

- wrap this up in a class and make it an executable that accepts command line arguments.
- provide better word matching that accounts for contractions. 
- write tests :)